plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -174,10 +174,16 @@
      * The OpenShift deploy timeout in seconds:
      * See this issue for background of why for end users on slow wifi on their laptops
      * DeploymentConfigs usually barf: https://github.com/openshift/origin/issues/10531
+     *
+     * Lets default to 2 hours which should be plenty of time on a slow connection to download
+     * all the images for a YAML/JSON manifest (which can contain many separate images)
      */
-    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds")
+    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds", defaultValue = "7200")
```

Comment:
Yeah - am hoping whatever we do will be a short term thing until its properly fixed. 

I guess for cases we're not building images with ImageStreams it might well just be better to go with `Deployment` anyway on OpenShift as I see DC`s as being much worse ;) The only real value of DCs seems to be when using ImageStreams really from what I can tell